src/rf/rf/urls.py

```diff
@@ -19,8 +20,11 @@
 urlpatterns = patterns(
     '',
     url(r'^', include(apps.home.urls)),
+    url(r'^', include(apps.core.urls)),
     url(r'^api/', include(router.urls)),
```

Comment:
For some endpoints, I think it would be cool to use a `.json` suffix instead of an `/api` prefix. So if we have a client-side route/view called `/users` that view would consume `/users.json`. I like the idea of being able to append `.json` to any client-side URL to view the raw data. This is just an idea I had that requires more thought though.